### ISSUE TITLE
Allow strings in classes (y values)

### ISSUE THREAD

**Issue by mfeurer** [automl/auto-sklearn#183]
A recent issue (#181) showed that auto-sklearn has issues with string values in the class labels (which scikit-learn supports). These should be transformed into integers prior to fitting.

**Comment by Aaaaaaare** [comment]
Are string values for classes now working for auto-sklearn, or do we still need to convert them to integer prior using auto-sklearn with datasets (multiclass classification)??

**Comment by mfeurer** [comment]
Nothing has changed yet.

**Comment by mfeurer** [comment]
We have added this functionality in the meantime - please reopen if it does not work.
